fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -1707,6 +1718,89 @@ private List<Tuple> queryLocal(Map<Tuple, Long> dataset, Comparisons.Type compar
         return hits.stream().map(Map.Entry::getKey).collect(Collectors.toList());
     }
 
+    static Stream<Arguments> simplePartitionConsolidationTest() {
+        return Stream.of(
+                // consolidate two low partitions into one
+                Arguments.of(2, 4, 3, new int[] {1, 1}, new int[] {2}),
+                // consolidate a partition into its previous neighbor
+                Arguments.of(2, 4, 3, new int[] {2, 2, 1}, new int[] {2, 3}),
+                // consolidate a partition falling between two partitions with capacity into its previous neighbor
+                Arguments.of(3, 7, 3, new int[] {5, 2, 5}, new int[] {7, 5}),
+                // consolidate a partition falling between two partitions which individually don't have enough
+                // capacity, but together they do
+                Arguments.of(3, 7, 3, new int[] {6, 2, 6}, new int[] {7, 7}),
+                // cannot consolidate a partition that has no neighbors with capacity
+                Arguments.of(3, 7, 3, new int[] {6, 2, 7}, new int[] {6, 2, 7}),
```

Comment:
It seems like you should have the same test starting with `{7, 2, 6}`

Reply:
Arguments.of(5, 10, 1, new int[] {7, 4, 10}, new int[] {7, 4, 10}),